editor.completion/src/org/netbeans/modules/editor/completion/DocumentationScrollPane.java

```diff
@@ -105,6 +106,16 @@ public DocumentationScrollPane(JTextComponent editorComponent) {
         
         // Add the completion doc view
         view = new HTMLDocView(bgColor);
+        Integer textZoom = (Integer) editorComponent.getClientProperty(TEXT_ZOOM_PROPERTY);
+        // Use the same logic as in o.n.editor.GlyphGutter.update().
```

Comment:
Shouldn't we use the fully-qualified class name here (no abbreviations)?

Reply:
Sure.

(The "o.n." thing seems to be a convention in NetBeans documentation, e.g. see  http://bits.netbeans.org/dev/javadoc/org-netbeans-modules-editor-lib/apichanges.html . But it's not consistently applied everywhere.)

Reply:
To me, personally, o.n (and o.n.m = org.netbeans.modules) seems quite obvious.

Reply:
@eirikbakke Go with @jlahoda's preference. He's been working on this codebase much longer than I :) I withdraw my objection.